tests/e2e/config/wdio.browser.conf.ts

```diff
@@ -0,0 +1,59 @@
+import { config as baseConfig } from './wdio.conf';
+import path from 'path';
+import url from 'url';
+import fs from 'fs/promises';
```

Comment:
As we're talking here:
https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/707#discussion_r1747089218

```suggestion
import path from 'node:path';
import url from 'node:url';
import fs from 'node:fs/promises';
```